packages/default/scss/notification/_layout.scss

```diff
@@ -28,13 +28,19 @@
             position: absolute;
             top: $notification-padding-y;
             right: $notification-padding-x;
+            cursor: pointer;
             font-size: inherit;
             line-height: inherit;
 
             &::before {
                 font-size: $icon-size;
             }
         }
+
+        > .k-notification-content {
+            display: inline-block;
+            vertical-align: middle;
```

Comment:
It appears that this `vertical-align` value is adequate for single line content, but doesn't look that good with multiline content:

![image](https://user-images.githubusercontent.com/90405/44411893-3fcb2c00-a570-11e8-8133-d3439880c0ff.png)

There's no design for that in kendo-ux at the moment.

Reply:
Checked with @MartinTsekov, the notification type icon will be always aligned vertically with the first row (vertical-align: top) of notification's content and with the 'close' button if it exists (this is not handled well in jquery if there is multiline notification). We also consider to make `.k-notification-wrap` `padding-top: 8px`, which does not breaks any rendering in jQuery, so that the notification has more space whithin. What do you think, @gyoshev, @joneff ?